target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-rutx11.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4018-rutx.dtsi"
+
+/ {
+	model = "Teltonika RUTX11";
+	compatible = "teltonika,rutx11";
+
+	memory {
```

Comment:
This is already in `qcom-ipq4018-rutx.dtsi`.

Reply:
Removed it.